src/Validation/CaValidation.php

```diff
@@ -59,3 +108,101 @@ public static function personId($check)
         throw new NotImplementedException(__d('localized', '%s Not implemented yet.'));
     }
 }
+
+class InvalidNumberRule
```

Comment:
I pushed it to my own fork. Didn't think it would trickle down...  I just needed something more robust and the regex was not working for some real area codes. You can ignore it... or is there something I can do to hide it? I'll look into that.